pkg/util/util.go

```diff
@@ -1238,6 +1238,17 @@ func GetFlagStringOrPromptPassword(cmd *cobra.Command, flag string) string {
 	return str
 }
 
+// GetBoolFlagPtr returns a pointer to the boolean flag value if set, or nil if not set
+func GetBoolFlagPtr(cmd *cobra.Command, flag string) *bool {
+	flagVal, _ := cmd.Flags().GetBool(flag)
+
+	if cmd.Flags().Changed(flag) {
+		return &flagVal
+	}
+
+	return nil
+}
```

Comment:
The `GetBool` should be inside the `if` statement. I think this is why the tests failed. 
Also, we should not ignore the error. Either return it or handle it in another way (log, etc.).

```suggestion
	if cmd.Flags().Changed(flag) {
	        flagVal, err := cmd.Flags().GetBool(flag)
        	if err != nil {
        	// handle error
 	        }
        	return &flagVal
	}
	return nil
}
```